catroid/src/createAtSchool/java/org/catrobat/catroid/common/FlavoredConstants.java

```diff
@@ -48,6 +48,7 @@
 	public static final String LIBRARY_BACKGROUNDS_URL_PORTRAIT = BASE_URL_HTTPS + "media-library/backgrounds-portrait";
 	public static final String LIBRARY_BACKGROUNDS_URL_LANDSCAPE = BASE_URL_HTTPS + "media-library/backgrounds-landscape";
 	public static final String LIBRARY_SOUNDS_URL = BASE_URL_HTTPS + "media-library/sounds";
+	public static final String CATROBAT_PRIVACY_POLICY_URL = "https://catrob.at/createatschoolprivacy";
```

Comment:
Thank you for adding this. However, when I changed the https://catrob.at/createatschoolprivacy short url just now to point to our main developer page instead of the privacy policy subpage (please try it out --- I'll change it back to the privacy page once this is fixed --- please remind me) and compiled the Create@School flavor: https://jenkins.catrob.at/job/Catroid/job/PR-3863/8/artifact/catroid/build/outputs/apk/createAtSchool/debug/catroid-createAtSchool-debug-PR-3863-8.apk --- the privacy policy button nevertheless still opens the main privacy policy page. Please fix.